process-automation/latest/model/processes/index.md

```diff
@@ -7,6 +7,10 @@ Processes are the collection of components that are used to build and represent
 There are three concepts associated with modeling processes to understand and differentiate between:
 
 * **Diagrams** are the container that process definitions are modeled in. A process diagram can contain multiple process definitions when [pools]({% link process-automation/latest/model/processes/bpmn.md %}#pools-and-lanes) are used to separate them.
+Process diagrams are colour coded with the following definitions:
```

Comment:
The following bullet points does not display indented

Reply:
Fixed cheers